llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -2542,6 +2542,12 @@ bool RISCVInstrInfo::verifyInstruction(const MachineInstr &MI,
         case RISCVOp::OPERAND_RTZARG:
           Ok = Imm == RISCVFPRndMode::RTZ;
           break;
+        case RISCVOp::OPERAND_VEC_POLICY:
+          Ok = (Imm & (RISCVII::TAIL_AGNOSTIC | RISCVII::MASK_AGNOSTIC)) == Imm;
```

Comment:
Can be simplified to `Imm <= (RISCVII::TAIL_AGNOSTIC | RISCVII::MASK_AGNOSTIC)`.

Reply:
Oops, it is a signed integer :(